locale/hu_HU/admin.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/hu_HU/admin.xml
+  *
+  * Copyright (c) 2013-2018 Simon Fraser University
+  * Copyright (c) 2003-2018 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  * Localization information: https://pkp.sfu.ca/wiki/index.php?title=Translating_OxS
+  * Localization strings.
+  -->
+
+<locale name="hu_HU" full_name="Magyar">
+	<message key="admin.hostedJournals">Tárolt folyóiratok</message>
+	<message key="admin.settings.journalRedirect">Folyóirat átirányítás</message>
+	<message key="admin.settings.options">Opciók</message>
+	<message key="admin.settings.journalRedirectInstructions">A főoldal át lesz irányítva erre a folyóiratra. Ez akkor hasznos ha az oldal pl. csak egy folyóiratot tartalmaz.</message>
+	<message key="admin.settings.noJournalRedirect">Ne irányítson át</message>
+	<message key="admin.settings.defaultMetricDescription">
```

Comment:
This is still in English. I suggest you either remove or translate this section.